homeassistant/components/eafm/station.py

```diff
@@ -0,0 +1,102 @@
+"""Support for a monitoring station."""
+from datetime import timedelta
+import logging
+
+from aioeafm import get_station
+import aiohttp
+
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.event import async_track_time_interval
+
+from .sensor import Measurement
```

Comment:
Platform details aren't allowed outside the platform.

Reply:
This module went away with DataUpdateCoordinator.